lib/models/task_list.dart

```diff
@@ -14,16 +14,18 @@ part 'task_list.g.dart';
 /// Here, the [id] is used to identify task lists throughout peers.
 @JsonSerializable(explicitToJson: true)
 class TaskList {
+  static const crdtMembers = ['isShared', 'title', 'sortBy'];
+
   String? id;
   bool isShared;
   String title;
   SortOption sortBy;
-  final List<Task>? elements;
+  final List<Task> elements;
 
   TaskList({
     this.id,
     required this.title,
-    this.elements,
+    this.elements = const [],
```

Comment:
It actually creates a constant list of size 0 that can not be changed (you also cannot add new elements). Still, this can be very confusing and I will change it.